### PR TITLE
Api Box update file

### PR DESCRIPTION
**Pull request by brandysmith**

### CODE REVIEW

File: ElasticBox/boxes-api.md [hunk 1]
```diff
@@ -1,73 +1,1770 @@
...
-**Network**
+![box3.png](../images/ElasticBox/box3.png)  ![box4.png](../images/ElasticBox/box4.png)
 
-| **Option**  |  **Description** |
-|----------|:-----|
-| Datacenter | Select a location to place the instance. |
-| Uplink Port Speed	| Optionally, upgrade the speed of the instance network port. By default, it is 10 Mbps. But you can upgrade to 100 or 1000 Mbps for private use, meaning no one else shares the network interface card port. |
+```
+[
+        {
```
Comment (srveit) [inline ElasticBox/boxes-api.md]: Change indet to 2 spaces.

File: ElasticBox/boxes-api.md [hunk 2]
```diff
@@ -1,73 +1,1770 @@
...
+    "name": "Redis",
+    "id": "7e846aa1-d490-4b8a-a2a1-555407a90105",
+    "members": [],
+    "owner": "public",
+    "organization": "public",
+    "events": {},
+    "draft_from": "aa5a019a-5dd6-4669-a591-ec52783b123e",
+    "icon": "images/platform/redis.png"
+  },
+{
```
Comment (srveit) [inline ElasticBox/boxes-api.md]: Indent 2 spaces.

File: ElasticBox/boxes-api.md [hunk 3]
```diff
@@ -1,73 +1,1770 @@
...
+
+        },
+        {
+            "required":true,
+            "type":"Port",
+            "name":"Port_variable",
+            "value":"80",
+            "visibility":"public"
+
+        },{
```
Comment (srveit) [inline ElasticBox/boxes-api.md]: Add newline after comma.

File: ElasticBox/boxes-api.md [hunk 4]
```diff
@@ -1,73 +1,1770 @@
...
+
+        },
+        {
+            "name":"Binding_db",
+            "type":"Binding",
+            "value":"2a7a5f6b-280d-47de-afaa-65db5dd85816",
+            "visibility":"public"
+
+        }
+        ],
```
Comment (srveit) [inline ElasticBox/boxes-api.md]: Indent this line and 8 following with 4 spaces.

File: ElasticBox/boxes-api.md [hunk 5]
```diff
@@ -1,73 +1,1770 @@
...
+    "workspace": "operations",
+    "created": "2015-10-28 13:47:15.564698"
+  },
+  "schema": "http://elasticbox.net/schemas/instance",
+  "id": "i-ywf1hu",
+  "icon": null
+}
+```
+### Example: Update a CloudFormation Stack in Real-Time
+ +
```
Comment (srveit) [inline ElasticBox/boxes-api.md]: Did you intend to have all of this lines start with `+`?

Comment (brandysmith) [inline ElasticBox/boxes-api.md]: No. They do not render that way in atom. How would I fix that?

Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: @brandysmith this particular line and the following does have a `+` in the text.  It should show up in Atom.  Please double-check

**Comment by brandysmith** [comment]
fix code block spacing

File: ElasticBox/boxes-api.md [hunk 6]
```diff
@@ -1,73 +1,1771 @@
...
+            "type":"Box"
+
+        },
+        {
+            "name":"Binding_db",
+            "type":"Binding",
+            "value":"2a7a5f6b-280d-47de-afaa-65db5dd85816",
+            "visibility":"public"
+
+  }
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: indent to match previous `{`

File: ElasticBox/boxes-api.md [hunk 7]
```diff
@@ -1,73 +1,1771 @@
...
+         "Type":"String",
+         "MinLength":"1",
+         "MaxLength":"255",
+         "AllowedPattern":"[\\x20-\\x7E]*",
+         "ConstraintDescription":"can contain only ASCII characters."
+      },
+    ...
+   }
+...
+...
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: double `...`

File: ElasticBox/boxes-api.md [hunk 8]
```diff
@@ -1,73 +1,1771 @@
...
+         "Type":"String",
+         "MinLength":"1",
+         "MaxLength":"255",
+         "AllowedPattern":"[\\x20-\\x7E]*",
+         "ConstraintDescription":"can contain only ASCII characters."
+      },
+    ...
+   },
+...
+...
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: double `...`

File: ElasticBox/boxes-api.md [hunk 9]
```diff
@@ -1,73 +1,1771 @@
...
+    "machines": [
+
+    ]
+  },
+  "tags": [
+
+  ],
+  "deleted": null,
+  "variables": [
+
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: either remove empty line or change to `variables: []`

File: ElasticBox/boxes-api.md [hunk 10]
```diff
@@ -1,73 +1,1771 @@
...
+  "created": "2015-10-28 13:47:15.567441",
+  "state": "processing",
+  "uri": "/services/instances/i-ywf1hu",
+  "boxes": [
+    {
+      "schema": "http://elasticbox.net/schemas/boxes/cloudformation",
+      "updated": "2015-10-28 13:46:55.247211",
+      "automatic_updates": "off",
+      "requirements": [
+
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: either remove empty line or change to `"requirements": []`

File: ElasticBox/boxes-api.md [hunk 11]
```diff
@@ -1,73 +1,1771 @@
...
+          "required": false,
+          "type": "Binding",
+          "name": "Binding",
+          "value": "5030fc49-773a-42fa-b569-c551e4f90d77",
+          "visibility": "private"
+        }
+      ],
+      "visibility": "workspace",
+      "members": [
+
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: either remove empty line or change to `"members": []`

File: ElasticBox/boxes-api.md [hunk 12]
```diff
@@ -1,73 +1,1771 @@
...
+      },
+      "owner": "operations",
+      "organization": "elasticbox",
+      "type": "CloudFormation Service",
+      "id": "91645eed-173f-4aac-a713-69c6be7582fe",
+      "name": "CF WordPress 2"
+    }
+  ],
+  "members": [
+
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: empty line

File: ElasticBox/boxes-api.md [hunk 13]
```diff
@@ -1,73 +1,1771 @@
...
+  ],
+  "bindings": [
+
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: empty line

File: ElasticBox/boxes-api.md [hunk 14]
```diff
@@ -1,73 +1,1771 @@
...
+   ],
+   "updated": "2015-10-28 13:48:24.367626",
+   "automatic_updates": "off",
+   "name": "CF WordPress 2",
+   "service": {
+     "type": "CloudFormation Service",
+     "id": "eb-5cn45",
+     "machines": [
+
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: empty line

File: ElasticBox/boxes-api.md [hunk 15]
```diff
@@ -1,73 +1,1771 @@
...
+     ]
+   },
+   "tags": [
+
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: empty line

File: ElasticBox/boxes-api.md [hunk 16]
```diff
@@ -1,73 +1,1771 @@
...
+   "deleted": null,
+   "policy_box": {
+     "profile": {
+       "location": "ap-northeast-1",
+       "schema": "http://elasticbox.net/schemas/aws/cloudformation/profile"
+     },
+     "updated": "2015-10-28 13:46:55.247211",
+     "automatic_updates": "off",
+     "requirements": [
+
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: empty line

File: ElasticBox/boxes-api.md [hunk 17]
```diff
@@ -1,73 +1,1771 @@
...
+     "provider_id": "7e841966-1dec-4460-a981-1db4e1eec10c",
+     "visibility": "workspace",
+     "template": {
+       "url": "/services/blobs/download/5630d1cf14841250525226a6/template.json",
+       "upload_date": "2015-10-28 13:46:55.186875",
+       "length": 104,
+       "content_type": "text/x-shellscript"
+     },
+     "members": [
+
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: empty line

File: ElasticBox/boxes-api.md [hunk 18]
```diff
@@ -1,73 +1,1771 @@
...
+       ],
+       "visibility": "workspace",
+       "template": {
+         "url": "/services/blobs/download/5630d61d14841250525226aa/simple_template.json",
+        "upload_date": "2015-10-28 13:46:55.186875",
+         "length": 104,
+         "content_type": "text/x-shellscript"
+       },
+       "members": [
+
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: empty line

File: ElasticBox/boxes-api.md [hunk 19]
```diff
@@ -1,73 +1,1771 @@
...
+       "organization": "elasticbox",
+      "schema": "http://elasticbox.net/schemas/boxes/cloudformation",
+       "type": "CloudFormation Service",
+       "id": "91645eed-173f-4aac-a713-69c6be7582fe",
+       "name": "CF WordPress 2"
+     }
+   ],
+   "schema": "http://elasticbox.net/schemas/instance",
+   "members": [
+
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: empty line

File: ElasticBox/boxes-api.md [hunk 20]
```diff
@@ -1,73 +1,1771 @@
...
+   ],
+   "updated": "2015-10-28 14:44:49.999798",
+   "automatic_updates": "off",
+   "name": "CF WordPress 2",
+   "service": {
+     "type": "CloudFormation Service",
+     "id": "eb-5cn45",
+     "machines": [
+
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: empty line

File: ElasticBox/boxes-api.md [hunk 21]
```diff
@@ -1,73 +1,1771 @@
...
+   "icon": null
+ }
+ ```
+ **Response**
+
+ ```
+ {
+   "box": "91645eed-173f-4aac-a713-69c6be7582fe",
+   "bindings": [
+
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: empty line

File: ElasticBox/boxes-api.md [hunk 22]
```diff
@@ -1,73 +1,1771 @@
...
+   "name": "CF WordPress 2",
+   "service": {
+     "type": "CloudFormation Service",
+     "id": "eb-5cn45",
+     "machines": [
+
+     ]
+   },
+   "tags": [
+
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: empty line

File: ElasticBox/boxes-api.md [hunk 23]
```diff
@@ -1,73 +1,1771 @@
...
+   "deleted": null,
+   "policy_box": {
+     "profile": {
+      "location": "ap-northeast-1",
+       "schema": "http://elasticbox.net/schemas/aws/cloudformation/profile"
+     },
+     "updated": "2015-10-28 13:46:55.247211",
+     "automatic_updates": "off",
+     "requirements": [
+
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: empty line

File: ElasticBox/boxes-api.md [hunk 24]
```diff
@@ -1,73 +1,1771 @@
...
+     "provider_id": "7e841966-1dec-4460-a981-1db4e1eec10c",
+     "visibility": "workspace",
+     "template": {
+       "url": "/services/blobs/download/5630d1cf14841250525226a6/template.json",
+       "upload_date": "2015-10-28 13:46:55.186875",
+       "length": 104,
+       "content_type": "text/x-shellscript"
+     },
+     "members": [
+
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: empty line

File: ElasticBox/boxes-api.md [hunk 25]
```diff
@@ -1,73 +1,1771 @@
...
+           "required": false,
+           "type": "Binding",
+          "name": "Binding",
+          "value": "5030fc49-773a-42fa-b569-c551e4f90d77",
+          "visibility": "private"
+        }
+      ],
+       "visibility": "workspace",
+       "members": [
+
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: empty line

File: ElasticBox/boxes-api.md [hunk 26]
```diff
@@ -1,73 +1,1771 @@
...
+       ],
+       "owner": "operations",
+      "id": "91645eed-173f-4aac-a713-69c6be7582fe",
+      "requirements": [
+
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: empty line

File: ElasticBox/boxes-api.md [hunk 27]
```diff
@@ -1,73 +1,1771 @@
...
+         "content_type": "text/x-shellscript"
+       },
+       "organization": "elasticbox",
+       "type": "CloudFormation Service",
+       "schema": "http://elasticbox.net/schemas/boxes/cloudformation"
+     }
+   ],
+   "schema": "http://elasticbox.net/schemas/instance",
+   "members": [
+
```
Comment (jim-phillips) [inline ElasticBox/boxes-api.md]: empty line

**Comment by srveit** [comment]
Closing as this file is a duplicate of ElasticBox/api-boxes.md

**Pull request closed without merging**